### PR TITLE
MF-360 MF-361

### PR DESCRIPTION
**Pull request by scottywakefield**

### CODE REVIEW

File: src/SFA.DAS.Reservations.Web.UnitTests/Employers/WhenCallingGetTransactorSignAgreement.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            EmployerReservationsController controller)
+        {
+            var result = controller.TransactorSignAgreement(routeModel) as ViewResult;
+
+            result.ViewName.Should().Be("TransactorSignAgreement");
+            var model = result.Model as SignAgreementViewModel;
+            model.BackRouteName.Should().Be(routeModel.PreviousPage);
+        }
+
+        //todo: employers
```
Comment (dashton82) [inline src/SFA.DAS.Reservations.Web.UnitTests/Employers/WhenCallingGetTransactorSignAgreement.cs]: is this still required?

File: src/SFA.DAS.Reservations.Web/Controllers/EmployerReservationsController.cs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
+using System.Reflection.Metadata.Ecma335;
```
Comment (dashton82) [inline src/SFA.DAS.Reservations.Web/Controllers/EmployerReservationsController.cs]: Is this needed?

File: src/SFA.DAS.Reservations.Web/Startup.cs [hunk 3]
```diff
@@ -54,15 +55,14 @@ public Startup(IConfiguration configuration, IHostingEnvironment environment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            IdentityModelEventSource.ShowPII = true;
```
Comment (dashton82) [inline src/SFA.DAS.Reservations.Web/Startup.cs]: Can probably remove this, or hide it behind only being visible for local and dev?

File: src/SFA.DAS.Reservations.Web/Controllers/EmployerReservationsController.cs (hunk 2 above)
Comment (scottywakefield) [reply src/SFA.DAS.Reservations.Web/Controllers/EmployerReservationsController.cs]: 🤢 sozzers. it's gone in pull/201.

File: src/SFA.DAS.Reservations.Web.UnitTests/Employers/WhenCallingGetTransactorSignAgreement.cs (hunk 1 above)
Comment (scottywakefield) [reply src/SFA.DAS.Reservations.Web.UnitTests/Employers/WhenCallingGetTransactorSignAgreement.cs]: employers is implemented in pull/201

File: src/SFA.DAS.Reservations.Web/Startup.cs (hunk 3 above)
Comment (scottywakefield) [reply src/SFA.DAS.Reservations.Web/Startup.cs]: will discuss with you

**Pull request merged**